test/WebJobs.Script.Tests.Integration/Host/StandbyManager/StandbyManagerE2ETests_Windows.cs

```diff
@@ -364,7 +364,7 @@ private async Task Verify_StandbyModeE2E_Java()
             // now specialize the host
             environment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "0");
             environment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteContainerReady, "1");
-            environment.SetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName, "java");
+            environment.SetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime, "java");
```

Comment:
Should this be `RpcWorkerConstants.JavaLanguageWorkerName` ?

Reply:
Leaving those. Addressed the other comments.